src/components/dashboard/FaceVerification/hooks/useVerificationAttempts.js

```diff
@@ -5,14 +5,24 @@ import GDStore, { useCurriedSetters } from '../../../../lib/undux/GDStore'
 export default () => {
   const store = GDStore.useStore()
   const verificationAttempts = store.get('verificationAttempts')
-  const [setVerificationAttempts] = useCurriedSetters(['verificationAttempts'])
-
-  const trackNewAttempt = useCallback(() => setVerificationAttempts(verificationAttempts + 1), [
-    setVerificationAttempts,
-    verificationAttempts,
+  const verificationAttemptErrMessages = store.get('verificationAttemptErrMessages')
```

Comment:
also pls adjust all vars/functions inside hook according to the new contract
to make theirs names shorter